samples/Sentry.Samples.Console.Customized/Program.cs

```diff
@@ -161,9 +161,9 @@ private static async Task Main()
                 const string msg = "{0} of {1} items we'll wait to flush to Sentry!";
                 SentrySdk.CaptureEvent(new SentryEvent
                 {
-                    LogEntry = new LogEntry
+                    Message = new SentryMessage
                     {
-                        Message = msg,
+                        Raw = msg,
```

Comment:
I agree the name isn't great, but I'd rather we stick to the C# name being closer to the protocol name.